jbi/queue.py

```diff
@@ -0,0 +1,315 @@
+"""This `queue` module stores Bugzilla webhook messages that we failed to sync
+to Jira.
+
+As Bugzilla sends us webhook messages, we want to eagerly accept them and
+return a `200` response so that we don't prevent it from sending new messages.
+But if we fail to sync a bug, we want to keep the message so we can retry it
+later. We also want to store any messages that might be successfuly synced, but
+were preceded by a message that wasn't synced.
+
+Classes:
```

Comment:
Is that a specific docstring format?

Reply:
Nope. I was trying to find if there was one. My goal was to document a "TL:DR" of the module + a high level look at the classes. If you have better formatting ideas, I'm all ears!

Reply:
This is good ! I was just curious